samples/detect.v1p3beta1.js

```diff
@@ -0,0 +1,198 @@
+/**
+ * Copyright 2018, Google,LLC.
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/* eslint-disable */
+
+'use strict';
+
+function localizeObjects(fileName) {
+  // [START vision_localize_objects]
+  // Imports the Google Cloud client libraries
+  const vision = require('@google-cloud/vision').v1p3beta1;
+  const fs = require('fs');
+
+  // Creates a client
+  const client = new vision.ImageAnnotatorClient();
```

Comment:
It goes like this:

```
–> imports
–> client
–> placeholder variables
–> request object
–> call api
–> print fields from the response
```

Please update to be:

```js
// Imports the Google Cloud client libraries
const vision = require('@google-cloud/vision').v1p3beta1;
const fs = require('fs');

// Creates a client
const client = new vision.ImageAnnotatorClient();

/**
 * TODO(developer): Uncomment the following line before running the sample.
 */
// const fileName = 'Local image file, e.g. /path/to/image.png';

const request = {
  image: {content: fs.readFileSync(fileName)},
};

client.
  objectLocalization.....
```

Please note the empty newlines separating `imports` | `client instantiation` | `placeholder variables` | `request objects` | `api call`

Please **never** deviate from this